posthog/api/instance_settings.py

```diff
@@ -69,10 +69,10 @@ def update(self, instance: InstanceSetting, validated_data: Dict[str, Any]) -> I
 
         if instance.key == "RECORDINGS_TTL_WEEKS":
 
-            if CLICKHOUSE_REPLICATION:
+            if MULTI_TENANCY:
```

Comment:
Not related to this PR but this is another setting I think we should simply deprecate